src/config/files.h

```diff
@@ -60,6 +60,7 @@
 #define DIR_DOWNLOADS "downloads"
 #define DIR_EDITOR    "editor"
 #define DIR_CERTS     "certs"
+#define DIR_PHOTOS    "photos"
```

Comment:
This is for avatars that are coming from the vcard yes?

Reply:
Yes it is